setup.py

```diff
@@ -5,22 +5,67 @@
 
 # -*- coding: utf-8 -*-
 
-import os
-
 import setuptools
+import subprocess
+
+from distutils.cmd import Command
```

Comment:
Looks like distutils is deprecated https://docs.python.org/3/library/distutils.html